mongodbatlasv2/api_legacy_backup.go

```diff
@@ -1,5 +1,5 @@
 /*
-API version: 2.0
```

Comment:
Side effect from having API version is that every regenerated file will contain version. 
I will remove version from template in the upcomming PRs to not contribute to large PR sizes.